content/docs/guides/gatsby.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Use with Gatsby - Flowbite React
+description: Learn how to install Flowbite React for your Gatsby project and start building websites with an open-source static site generator built on top of React and GraphQL
+---
+
+## Create project
+
+1. Run the following command to create a new Gatsby project with Tailwind CSS:
+
+```bash
+npm init gatsby
+```
+
+2. Select `"Tailwind CSS"` to `"Would you like to install a styling system?"` question.
+
+## Install Flowbite React
+
+1. Run the following command to install `flowbite` and `flowbite-react`:
+
+```bash
+npm i flowbite flowbite-react
```

Comment:
I believe we can safely remove the `flowbite` part of `npm i` for all of these guides, because it is an implicit dependency of `flowbite-react`

Reply:
agreed